tests/rpmpgpcheck.c

```diff
@@ -1,12 +1,12 @@
 #include <stddef.h>
-#include <assert.h>
 #include <limits.h>
 #include <stdio.h>
 
 #include <unistd.h>
 #include <sys/mman.h>
 
 #include <rpm/rpmpgp.h>
+#include "../rpmio/digest.h"
```

Comment:
This seems unnecessary and kinda harmful, I'd rather test programs only use public APIs.